acceleration/README.md

```diff
@@ -0,0 +1,168 @@
+# Sample
+
+This sample showcases the way to use spiced to simplify the process of building a CQRS pattern for your application.
+
+In this sample, we have a `/orders` API which creates a random order with a random count.
+
+The sample uses a single postgres database to receive order command and store products, in MicroServices reaility, they could be in separate database instances which make the sample even more appealing.
+
+There are 3 `/rankings` API to show the top 5 selling products:
+- /rankings - fetch data from postgres
+- /rankings-accelerated - fetch data from spiced runtime without materialized ranking view, it pulls data from postgres into edge acceleration to improve query performance
+- /rankings-materialized - fetch data from spiced runtime with materialized ranking view, it pulls data from ranking view from the previous spiced runtime and accelerates it locally
+
+## How to run
+
+`docker-compose up --build`
+
+then call the rankings API
+`curl localhost:9999/rankings`
+
+## Performance benchmark using oha
```

Comment:
It would be good to have a summary of the results in a table format - as it is, its hard to parse the details quickly.

Reply:
Please resolve / address in next iteration.  Closing comment to merge.